packages/server/lambdas/alfresco/list-nodes.ts

```diff
@@ -16,9 +16,10 @@ const getNodes = async (id: string, type?: string) => {
     if (type && type === 'file') {
       queryParameter = `?where=(isFile=true)`;
     }
-    return await axios.get(`${alfrescoCoreAPIUrl}/nodes/${id}/children${queryParameter}`);
+    const response = await axios.get(`${alfrescoCoreAPIUrl}/nodes/${id}/children${queryParameter}`, options);
+    return response;
   } catch (error) {
-    return error;
+    log.error(error);
```

Comment:
No complaints from linter or anything that we're not returning anything here? One possibility would be to remove the try-catch here and let any errors be caught in the handleRequest try-catch. This is something that probably should be applied elsewhere as well... 🤔